newrelic-api/src/main/java/com/newrelic/api/agent/HttpParameters.java

```diff
@@ -91,6 +109,26 @@ public String getProcedure() {
         return procedure;
     }
 
+    /**
+     * Returns the HTTP status code for the call.
+     *
+     * @return the status code for the call, null if not available
+     * @since 6.5.0
```

Comment:
Is this still going to be `6.5.0`?

Reply:
nope. This is how long ago I started working on this.